client/service/src/builder.rs

```diff
@@ -530,10 +531,12 @@ where
 	};
 
 	let (rpc, socket_addr) = start_rpc_servers(&config, gen_rpc_module, rpc_id_provider)?;
+	let mut multiaddr: Multiaddr = socket_addr.ip().into();
```

Comment:
Do you have, at hand, how does `Multiaddr` look like at this stage?

Reply:
`/ip4/127.0.0.1`